lib/improver/threshold.py

```diff
@@ -272,9 +272,12 @@ def process(self, input_cube):
         # Force the metadata to temporary conventions
         if self.below_thresh_ok:
             cube.attributes.update({'relative_to_threshold': 'below'})
+            cube.rename(
+                "probability_of_{}_below_threshold".format(cube.name()))
         else:
             cube.attributes.update({'relative_to_threshold': 'above'})
-        cube.rename("probability_of_{}".format(cube.name()))
+            cube.rename(
+                "probability_of_{}_above_threshold".format(cube.name()))
```

Comment:
The ```Returns``` section of the ```process``` doc string needs to be updated to reflect the changes that have been made here.

Reply:
Updated